GeneratedRegexPatterns/HighConfidenceSecurityModels.json

```diff
@@ -11,6 +12,7 @@
   {
     "Id": "SEC101/156",
     "Name": "AadClientAppIdentifiableCredentials",
+    "Label": "an Azure Entra client app password",
```

Comment:
Ignore this, testing if VS Code in browser works on tis repo.

Reply:
Woohoo, it does!

Reply:
Amazing. Awesome.